ansible/group_vars/all.yml

```diff
@@ -38,7 +38,11 @@ insecure_registrys:
 #no_proxy: "127.0.0.1,localhost,docker-registry.somecorporation.com"
 
 # The port that the Kubernetes apiserver component listens on.
-kube_master_api_port: 443
+kube_api_secure_port: 443
```

Comment:
This actually breaks my CI as I am changing the value of ``kube_master_api_port``. Not necessarly breaking the ansible playbook itself. Though,  can imagine someone else could by relying on existence of this variable.

In general, it is not a good idea to change names of variables once they are released into the wild. Port ``443`` is a secure port it makes sense to name it as ``kube_api_secure_port``. Still, the apiserver can be run on 8080 port, so the ``master_api_port`` is still general enough to cover both secure and unsecure ports.